### PR TITLE
Add model id to all `exists()` method calls

### PR DESCRIPTION
**Pull request by bancer**
Related issue - https://github.com/dereuromark/cakephp-shim/issues/25
Also see https://github.com/dereuromark/cakephp-shim/blob/2.x/Model/ShimModel.php#L558.

### CODE REVIEW

**Comment by dereuromark** [comment]
Tests need to be adjusted: https://travis-ci.org/cakephp/cakephp/jobs/448237264

**Comment by bancer** [comment]
@dereuromark Done :).

File: lib/Cake/Console/Templates/default/actions/controller_actions.ctp [hunk 1]
```diff
@@ -132,7 +132,7 @@
  */
 	public function <?php echo $admin; ?>delete($id = null) {
 		$this-><?php echo $currentModelName; ?>->id = $id;
-		if (!$this-><?php echo $currentModelName; ?>->exists()) {
+		if (!$this-><?php echo $currentModelName; ?>->exists($id)) {
```
Comment (markstory) [inline lib/Cake/Console/Templates/default/actions/controller_actions.ctp]: Do we need to set the property on line 134 still?

Comment (bancer) [reply lib/Cake/Console/Templates/default/actions/controller_actions.ctp]: I believe it is still needed for `delete()` call below.

Comment (dereuromark) [reply lib/Cake/Console/Templates/default/actions/controller_actions.ctp]: It could be moved down to it then maybe.

Comment (bancer) [reply lib/Cake/Console/Templates/default/actions/controller_actions.ctp]: Done.

**Pull request merged**